metrics-datadog/src/main/java/org/coursera/metrics/datadog/transport/UdpTransport.java

```diff
@@ -22,6 +24,7 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(UdpTransport.class);
   private final StatsDClient statsd;
+  private final HashMap lastSeenCounters = new HashMap<String,Integer>();
```

Comment:
Hrm. Looks like nobody's releasing new versions of this library right now. The most recent released version is [2.0.8](https://github.com/indeedeng/java-dogstatsd-client/blob/java-dogstatsd-client-2.0.8/src/main/java/com/timgroup/statsd/StatsDClient.java#L40) and it's still an `int`, but master is [a long](https://github.com/indeedeng/java-dogstatsd-client/blob/master/src/main/java/com/timgroup/statsd/StatsDClient.java#L40).

I'll open an issue over there.